manual/C/Help_ch_GUIMenus.xml

```diff
@@ -3264,7 +3264,9 @@
         <para>Appearance of the Account Register Display is highly customizable (see
           <ulink
 url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.html#txns-regstyle1">
-          Tutorial and Concepts Guide, Choosing a Register Style</ulink>).
+          Tutorial and Concepts Guide, Choosing a Register Style</ulink>)  and <ulink
+url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.html#txns-columnwidths1">Setting
```

Comment:
Please use the entity &url-docs-build-C;

@jralls For now the crosslinks go to the build server because that are more recent. OTOH code has more downtime than www.
Or we expand the link to something like `<ulink … code/…><footnote>If that fails, try <ulink … www/…></footnote>`

Reply:
I prefer to link to the official documentation in the www according to the release.
Since the versions are released every three months, the timeliness of the documentation is not crucial.

Reply:
I don't object to change it—after the release.

Reply:
The release manager can now swwitch ENTITY url-docs  from "&url-docs-draft;" to "&url-docs-release;"
https://wiki.gnucash.org/wiki/index.php?title=Documentation_Release_Process&type=revision&diff=21109&oldid=17299